tools/unix/generate_localizations.sh

```diff
@@ -1,20 +1,36 @@
 #!/bin/bash
```

Comment:
У меня вот такой warning выдается каждый раз:

```
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
```